spec/models/emotion_spec.rb

```diff
@@ -28,6 +28,17 @@
     it 'the same emotion word should not be used' do
       new_word = FactoryBot.build(:emotion, word: emotion.word)
       expect(new_word).to_not be_valid
-   end
+    end
+
+    it "should lowercase words" do
+      emotion = Emotion.new(word: "Enthusiastic")
+      emotion.save!
+      expect(emotion.word).to include("enthusiastic")
+    end
+
+    it "the capitalized emotion word should not be used" do
+      new_word = FactoryBot.build(:emotion, word: emotion.word.upcase!)
+      expect(new_word).to_not be_valid
+    end
```

Comment:
If `emotion.word` will be forced to downcase each time on create or save, maybe we don't need this validation.